packages/components/src/Combobox/components/ComboboxTrigger/ComboboxTriggerChip/ComboboxTriggerChip.css

```diff
@@ -1,10 +1,11 @@
 .trigger {
   display: inline-flex;
-  gap: var(--space-small);
   padding: var(--space-small) var(--space-base);
   border: var(--border-base) solid rgb(230, 234, 235);
```

Comment:
this isn't happening in this PR but where is this random RGB value coming from? Can we use one of our system colors?

Reply:
Looks like `color-border` would work here

Reply:
I feel like I saw this rgb value in the figma at some point but `color-border` does the trick!